src/index.js

```diff
@@ -159,6 +159,8 @@ class AvatarEditor extends React.Component {
     borderRadius: PropTypes.number,
     width: PropTypes.number,
     height: PropTypes.number,
+    canvasWidth: PropTypes.number,
```

Comment:
I think they should always come together, so maybe something like this would be better?
`
canvasSize: PropTypes.shape({
      height: PropTypes.number.isRequired,
      width: PropTypes.number.isRequired,
)}
`